apps/tree-shaking-tester/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # @repo/tree-shaking-tester
 
+## 0.1.0
+
+### Minor Changes
+
+- 9d47b7d: new icon
```

Comment:
let's also remove it from here (and from other changelogs), i think this should be added only to Icons package changelog